java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -981,19 +1162,418 @@ public static HostColumnVector timestampNanoSecondsFromBoxedLongs(Long... values
     return build(DType.TIMESTAMP_NANOSECONDS, values.length, (b) -> b.appendBoxed(values));
   }
 
+  /**
+   * WARNING: Debug only method to print a passed in buffer
+   */
+  private static void printBuffer(HostMemoryBuffer buffer) {
```

Comment:
I think this is a duplicate with the other one that should both probably be a part of HostMemoryBuffer.

Reply:
I think now we have only one printBuffer method.